modules/control-utility/src/main/java/org/apache/ignite/internal/commandline/checkpointing/CheckpointAbstractSubCommand.java

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright 2022 GridGain Systems, Inc. and Contributors.
+ *
+ * Licensed under the GridGain Community Edition License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.gridgain.com/products/software/community-edition/gridgain-community-edition-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.internal.commandline.checkpointing;
+
+import java.util.Collection;
+import java.util.logging.Logger;
+import java.util.stream.Collectors;
+import org.apache.ignite.internal.client.GridClient;
+import org.apache.ignite.internal.client.GridClientCompute;
+import org.apache.ignite.internal.client.GridClientConfiguration;
+import org.apache.ignite.internal.client.GridClientDisconnectedException;
+import org.apache.ignite.internal.client.GridClientNode;
+import org.apache.ignite.internal.commandline.AbstractCommand;
+import org.apache.ignite.internal.commandline.Command;
+import org.apache.ignite.internal.commandline.CommandLogger;
+import org.apache.ignite.internal.commandline.checkpointing.subcommands.CheckpointForceCommand;
+import org.apache.ignite.internal.dto.IgniteDataTransferObject;
+import org.apache.ignite.internal.util.typedef.F;
+import org.apache.ignite.internal.visor.VisorTaskArgument;
+
+/**
+ * Base cli command class for {@link CheckpointForceCommand }
+ *
+ * Launch checkpointing process on a cluster and return results
+ */
+public abstract class CheckpointAbstractSubCommand<
+    MetadataArgsDto extends IgniteDataTransferObject,
+    MetadataResultDto extends IgniteDataTransferObject>
+    extends AbstractCommand<MetadataArgsDto> {
+
+    /**
```

Comment:
Please, add meaningful Javadoc for methods and fields in accordance with the code style:
```
Every method, field or initializer public, private or protected in top-level, inner or anonymous type should have at least minimal Javadoc comments including description and description of parameters using @param, @return and @throws Javadoc tags, where applicable.
```

And use one-line comments where applicable:
```
    /** Command arguments. */
    private MetadataArgs args;
```